cylc/flow/glbl_cfg_lazy.py

```diff
@@ -18,5 +18,5 @@
 
 def glbl_cfg(cached=True):
     """Load and return the global configuration singleton instance."""
-    from cylc.flow.cfgspec.globalcfg import GlobalConfig
+    from cylc.flow.config_schema import GlobalConfig
     return GlobalConfig.get_inst(cached=cached)
```

Comment:
Shouldn't this file be kept in the `cfgspec` directory/module? Or is that module going to be removed later?

Reply:
Has `cfgspec` not been removed yet?

Reply:
It has been.